### PR TITLE
Extole Partnership Docs - MERGE 5/9

### PR DESCRIPTION
**Pull request by KellieHawks**
@ekthayer
Updated submitted documentation to align with other Braze documentation.
Your documentation will be added to the site once we get the go-ahead from our partnership team.
Please feel free to reach out if you have any issues.

Thanks,
The Braze Documentation Team

In reference to this closed PR https://github.com/braze-inc/braze-docs/pull/3481

### CODE REVIEW

File: _docs/_partners/message_orchestration/channel_extensions/loyalty/extole.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+With the Braze and Extole integration, you can pull customer events and attributes from Extole refer-a-friend and growth programs into Braze, empowering you to create more personalized marketing campaigns that boost customer acquisition, engagement, and loyalty. You can also dynamically pull Extole content attributes, such as personalized share codes and links, into Braze communications.
+
+## Prerequisites
+
+| Requirement | Description |
+| ----------- | ----------- |
+| Extole account | An Extole account is required to take advantage of this partnership. |
+| Braze REST API key | A Braze REST API key with `users.track` permissions. <br><br> This can be created within the **Braze Dashboard > Developer Console > REST API Key > Create New API Key**. |
+| Braze REST test API key (optional) | A test API key that can be used for testing purposes if you’d like these requests sent to a separate staging Braze instance. |
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/channel_extensions/loyalty/extole.md:23]: I'm not sure putting "test" between "REST" and "API" is grammatically correct 🤔  
```suggestion
| Braze test REST API key (optional) | A test API key that can be used for testing purposes if you’d like these requests sent to a separate staging Braze instance. |
```

File: _docs/_partners/message_orchestration/channel_extensions/loyalty/extole.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+2. Enter a name for the key and select **Webhook** as the key type. 
+3. Add your Braze REST API key in the **Partner Key ID** field, select `HTTP_BASIC` as the **Algorithm**, and click **Create Key**.<br><br>![][4]{: style="max-width:80%;"}
+
+Next, work with your Extole success or implementation manager to create a new webhook. They will configure the webhook for you using your newly generated key and Braze instance URL.<br><br>![][5]{: style="max-width:80%;"}
+
+## Customization
+
+### Configure a staging API key for testing
+
+If you only provide one Braze REST API key to Extole, only production events will be sent to Braze. If you also want to send staging/testing events, create another Braze REST API key and configure an additional webhook integration in the Extole dashboard.
```
Comment (bre-fitzgerald) [inline _docs/_partners/message_orchestration/channel_extensions/loyalty/extole.md:88]:
```suggestion
If you only provide one Braze REST API key to Extole, only production events will be sent to Braze. If you also want to send staging or testing events, create another Braze REST API key and configure an additional webhook integration in the Extole dashboard.
```

**Review by bre-fitzgerald — CHANGES_REQUESTED** [review]
Looks great! Left a few small suggestions for you :)

File: _docs/_partners/message_orchestration/channel_extensions/loyalty/extole.md (hunk 1 above)
Comment (ekthayer) [reply _docs/_partners/message_orchestration/channel_extensions/loyalty/extole.md]: Definitely a typo, thank you for pointing this out!!!

File: _docs/_partners/message_orchestration/channel_extensions/loyalty/extole.md (hunk 2 above)
Comment (ekthayer) [reply _docs/_partners/message_orchestration/channel_extensions/loyalty/extole.md]: Looks good to me!

**Review by ekthayer — DISMISSED** [review]

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**